tests/functional/finetune/finetune_functional_test.py

```diff
@@ -47,7 +47,7 @@ def run_input_map(request):
 def list_input_map(request):
     return request.param
 
-
+'''
```

Comment:
Can you mark this test as skipped?

Reply:
@mikelam-us-aixplain, to remove AWS ASR from the tests, you should erase AWS ASR entry from the `finetune_test_run_data.json` file. It is the second element: https://github.com/aixplain/aiXplain/blob/M-5283031693-Removing-Problematic-Finetune_Test/tests/functional/finetune/data/finetune_test_run_data.json#L7C22-L8
Please also save a copy of the file with all suppliers as a backup.

Reply:
@lucas-aixplain why remove when this can be marked as skipped? We can leave a note to why it was marked as skipped.